pkg/cmd/edge_applications/publish/publish.go

```diff
@@ -109,6 +110,7 @@ func (cmd *PublishCmd) run(f *cmdutil.Factory) error {
 	}
 
 	typeLang := gjson.Get(string(file), "type")
+	fmt.Println(Path)
```

Comment:
Hey dude, Do we need to use this print here?

Reply:
Good catch! I'll remove this right away.